themes/theme-ios11/pages/Cart/components/Item/components/Coupon/components/Layout/components/Code/index.jsx

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import I18n from '@shopgate/pwa-common/components/I18n';
+import { Ellipsis } from '@shopgate/engage/components';
+import { css } from 'glamor';
+
+/**
+ * on iOS theme the maximum with the
+ * ellipsis wrapper can have is almost 179px
+ * @type {{ellipsis: string}}
+ */
+const styles = {
+  ellipsis: css({
+    width: '175px',
```

Comment:
both theme should use the same styling since both can run on any device.
also make sure it works on large devices like tablets